CHANGELOG.md

```diff
@@ -16,6 +16,53 @@ Mapbox welcomes participation and contributions from everyone.
 ## Dependencies
 * Update mapbox-gestures-android dependency to [v0.8.0](https://github.com/mapbox/mapbox-gestures-android/releases/tag/v0.8.0). ([1645] (https://github.com/mapbox/mapbox-maps-android/pull/1645))
 
+# 10.8.0 September 7, 2022
+## Features ✨ and improvements 🏁
+* Introduce a callback to be invoked when the device compass sensors need to be re-calibrated. ([1513](https://github.com/mapbox/mapbox-maps-android/pull/1513))
+* Add support for `LocationComponentSettingsInterface.pulsingMaxRadius` to follow location's accuracy radius. ([1561](https://github.com/mapbox/mapbox-maps-android/pull/1561))
+* Avoid map content disappearing on the sides of the screen when LOD is enabled ([1650](https://github.com/mapbox/mapbox-maps-android/pull/1650))
+* Enable [pitch] and [distance-from-center] style expressions ([1650](https://github.com/mapbox/mapbox-maps-android/pull/1650))
```

Comment:
```suggestion
* Enable [pitch] and [distance-from-center] when using raw style expressions. ([1650](https://github.com/mapbox/mapbox-maps-android/pull/1650))
```

Reply:
these expressions are not exposed yet, need to remove